diff --git a/pyproject.toml b/pyproject.toml
index d2dd76a..bcdbc3c 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -29,8 +29,8 @@ transformers = [
   {markers = "sys_platform == 'darwin' and platform_machine == 'x86_64'", version = "~4.42.0"}
 ]
 numpy = [
-    { version = ">=1.24.4,<3.0.0", markers = 'sys_platform != "darwin" or platform_machine != "x86_64"' },
-    { version = ">=1.24.4,<2.0.0", markers = 'sys_platform == "darwin" and platform_machine == "x86_64"' },
+    { version = ">=1.22.0,<3.0.0", markers = 'sys_platform != "darwin" or platform_machine != "x86_64"' },
+    { version = ">=1.22.0,<3.0.0", markers = 'sys_platform == "darwin" and platform_machine == "x86_64"' },
 ]
 jsonlines = "^3.1.0"
 Pillow = "^10.0.0"
