diff --git a/pyproject.toml b/pyproject.toml
index 60d94eb..4698eac 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -24,14 +24,16 @@ packages = [
 python = "^3.10"
 torch = "^2.2.2"
 torchvision = "^0"
-numpy = [
-    { version = "^2.1.0", markers = 'python_version >= "3.13"' },
-    { version = "^1.24.4", markers = 'python_version < "3.13"' },
-]
+numpy = "*"
+# numpy = [
+#     { version = "^2.1.0", markers = 'python_version >= "3.13"' },
+#     { version = "^1.24.4", markers = 'python_version < "3.13"' },
+# ]
 jsonlines = "^3.1.0"
 Pillow = "^10.0.0"
 tqdm = "^4.64.0"
-mean_average_precision = "^2021.4.26.0"
+mean_average_precision = ">=2021.4.26.0"  # See https://github.com/DS4SD/docling-ibm-models/pull/52
+
 opencv-python-headless = "^4.6.0.66"
 huggingface_hub = ">=0.23,<1"
 
