diff --git a/pyproject.toml b/pyproject.toml
index e5ce49e..de935ed 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -24,10 +24,7 @@ packages = [
 python = "^3.9"
 torch = "^2.2.2"
 torchvision = "^0"
-numpy = [
-    { version = "^2.1.0", markers = 'python_version >= "3.13"' },
-    { version = "^1.24.4", markers = 'python_version < "3.13"' },
-]
+numpy = "*"
 jsonlines = "^3.1.0"
 Pillow = "^10.0.0"
 tqdm = "^4.64.0"
